test/archethic/contracts/interpreter/action_interpreter_test.exs

```diff
@@ -515,6 +545,57 @@ defmodule Archethic.Contracts.Interpreter.ActionInterpreterTest do
       assert %Transaction{data: %TransactionData{content: "hello"}} = sanitize_parse_execute(code)
     end
 
+    test "should be able to use a function" do
```

Comment:
This test is already done in interpreter_test so I think you can delete this one

Same for other tests that uses FunctionInterpreter